doc/man3/EVP_EncryptInit.pod

```diff
@@ -462,6 +476,51 @@ Due to the constraints of the API contract of this function it shouldn't be used
 in applications, please consider using EVP_CipherUpdate() and
 EVP_CipherFinal_ex() instead.
 
+=item EVP_CIPHER_can_pipeline()
+
+This function checks if a B<EVP_CIPHER> fetched using EVP_CIPHER_fetch() supports
+cipher pipelining. If the cipher supports pipelining, it returns 1, otherwise 0.
+This function will return 0 for non-fetched ciphers such as EVP_aes_128_gcm().
+There are currently no built-in ciphers that support pipelining.
+
+Cipher pipelining support allows an application to submit multiple chunks of
+data in one set of EVP_CipherUpdate()/EVP_CipherFinal calls, thereby allowing
+the provided implementation to take advantage of parallel computing. This is
+beneficial for hardware accelerators as pipeline amortizes the latency over
+multiple chunks. For TLS, this provides the ability for reading and writing
+multiple records in parallel.
```

Comment:
Strictly speaking we can't do this yet for TLS. IIRC the original intent was for there to be a follow on PR to make the necessary changes to libssl to use this new API. So perhaps drop this last sentence for now, and re-add it when TLS support is added.